database/database.rs

```diff
@@ -201,6 +202,7 @@ impl<D> Database<D> {
 
 impl Database<WALClient> {
     const STATISTICS_UPDATE_INTERVAL: Duration = Duration::from_millis(100);
+    const CHECKPOINT_INTERVAL: Duration = Duration::from_secs(60);
```

Comment:
Every minute might be too often? Though you can load a lot of data in that time.

Reply:
Oh can we move both of these to `constants.rs` please!